transcriptic/cli.py

```diff
@@ -179,6 +179,68 @@ def cli(ctx, api_root, email, token, organization, config):
             pass
 
 
+@cli.command("exec")
```

Comment:
we probably want to limit this by feature group. See `@cli.command("submit", ...)`